src/core/qgsapplication.h

```diff
@@ -449,6 +449,12 @@ class CORE_EXPORT QgsApplication : public QApplication
      */
     static QString locale();
 
+    /**
+     * Sets the QGIS locale.
```

Comment:
This needs some updates to clarify that this is mostly there for custom applications, whereas in QGIS this is internally triggered by the application already.

Reply:
Is it clear enough now?

Reply:
Yep, looks good